### ISSUE TITLE
getLatestStableBrowsers returns array of all browsers

### ISSUE THREAD

**Issue by myrddraall** [Nyalab/caniuse-api#56]
getLatestStableBrowsers is returning  an array  with 180 elements

```["firefox 2", "firefox 3" ...]```

instead of the documented

```[ 'safari 8',
  'opera 26',
  'ios_saf 8.1',
  'ie_mob 11',
  'ie 11',
  'firefox 33',
  'chrome 39' ]
```

**Comment by natotthomer** [comment]
I would second this. Having both would be very useful.

**Comment by chrisdemars** [comment]
Same happening to me. I just need the latest version.

**Comment by cacheflow** [comment]
I found the problem and was able to fix it. https://github.com/Nyalab/caniuse-api/pull/63

**Comment by ooflorent** [comment]
Fixed by #71
